Gruntfile.js

```diff
@@ -6,6 +6,10 @@ module.exports = function(grunt) {
       build: {
         src  : ['./src/*.js', './src/**/*.js'],
         dest : './build/ros3d.js'
+      },
+      build_esm: {
+        src  : [ './src-esm/ESMHeader.js', './build/ros3d.js', './src-esm/ESMFooter.js'],
```

Comment:
Not required but might be more concise to keep all code in `./src` directory rather than creating a new one. Do you need a new directory to avoid a conflict with the existing grunt job for instance?

Reply:
Nope, `src-esm` will be removed. I was toying with the idea of wrapping the original src instead of rewriting it. I imagine we should just move to writing ES6 only now?